test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -273,21 +273,16 @@ public static String getOSSpecificInstructions() {
                 FN+Up Arrow/FN+Down Arrow - Deselect current selection;
                                    scroll up/down one JViewport view;
                                    first visible cell in current column gets focus
-                Control-FN+Up Arrow/FN+Down Arrow - Deselect current selection;
-                                                    move focus and view to
-                                                    first/last cell in current row
-                F2 - Allows editing in a cell containing information without
+                fn - Allows editing in a cell containing information without
                      overwriting the information
                 Esc -  Resets the cell content back to the state it was in
                        before editing started
-                Ctrl+A, Ctrl+/ - Select All
-                Ctrl+\\ - Deselect all
+                Cmd+A - Select All
                 Shift-Up/Down Arrow -  Extend selection up/down one row
                 Shift-Left/Right Arrow - Extend selection left/right one column
-                FN-Shift Up/Down Arrow -  Extend selection to top/bottom of column
-                Shift-PageUp/PageDown - Extend selection up/down one view and scroll
-                                        table
-                                """;
+                Ctrl-Shift Up/Down Arrow -  Extend selection to top/bottom of row
+                Ctrl-Shift Left/Right Arrow -  Extend selection to first/last of column
```

Comment:
Its not the extension of one cell (Its actually the number of cells in that row/column) and also extend selection for selected cell would not be right phrase, I feel.

Reply:
I mean the selection is extended relative to that particular cell. When `Shift + UP` arrow is pressed, the cell just above the current selected cell is selected in that column and vice-versa. Same applied to `Shift + Left/Right` arrow but in row order.

And with `CTRL + Shift + UP / DOWN / LEFT / RIGHT` key combination is pressed, it will select the entire items just `above / below / left / right` of current selected item in column or row order.

Current instruction says `Shift-Up/Down Arrow -  Extend selection up/down one row` , but selection is limited to one cell only not entire row.

So, I thought the change will convey the instructions in a better way.

Reply:
Yeah true, but then instruction will be lengthy so better to leave it that way I guess.